projects/js-packages/shared-extension-utils/src/block-icons.js

```diff
@@ -1,25 +1,10 @@
-import colorStudio from '@automattic/color-studio';
```

Comment:
I also checked that `@automattic/color-studio` is still used by other features in this package so we can keep it in `package.json`.